components/onboarding_tour/channels_and_direct_messages_tour_tip.tsx

```diff
@@ -57,7 +55,7 @@ export const ChannelsAndDirectMessagesTour = ({firstChannelName}: Props) => {
                 <FormattedMarkdownMessage
                     id='onboardingTour.ChannelsAndDirectMessagesTour.townSquare'
                     defaultMessage={'We’ve also added the **{townSquare}** and **{offTopic}** channels for everyone on your team.'}
-                    values={{townSquare: toTitleCase(townSquareDisplayName), offTopic: toTitleCase(offTopicDisplayName)}}
+                    values={{townSquare: townSquareDisplayName, offTopic: offTopicDisplayName}}
```

Comment:
Since these are already the display names of these channels, I don't think we should convert them to title case because `.display_name` should be authoritative for display.

Reply:
@matthewbirtch Thoughts?

Reply:
Channels should display however the user has named them in whatever casing they've chosen for the display name